notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    endpoint = vertex_ai.Endpoint.list(
</pre><p>Instead of getting endpoint from the list, just use the return value from <code>create_endpoint</code>.</p><p>The current method is prone to flakiness as your expected endpoint may not be the last one listed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
Yep makes sense. Looks like the SDK now has built-in methods so will use those directly to retrieve the endpoint without listing first.